### DISCUSSION TITLE
Not having tasks in the day planner

### DISCUSSION THREAD

**Discussion by ShemJM** [ivan-lednev/obsidian-day-planner#423] [General]
Hey, I just setup the day planner in my vault. It defaults to making everything a task, which is not really how I intend to use the plugin. I would like to use it more as a time tracker. So at the end of the day, I will fill it in and say how much time I spent working or at the gym on a personal project for example. 

I might add [[personal project A]]  1200-1700 and I would like that to link back the time to the relevant project note, so that I can see every day that I worked on it and how much time I have spent there. 

Is that possible?

**Comment by ivan-lednev** [comment]
Time tracking is not implemented, it's tracked here: #330
